.prettierrc.json

```diff
@@ -1,12 +1,12 @@
 {
   "arrowParens": "always",
-  "bracketSameLine": true,
+  "bracketSameLine": false,
   "bracketSpacing": true,
   "embeddedLanguageFormatting": "auto",
   "endOfLine": "lf",
   "htmlWhitespaceSensitivity": "css",
   "insertPragma": false,
-  "jsxSingleQuote": false,
+  "jsxSingleQuote": true,
```

Comment:
<details>

_for my own reference_

You proposed a change to [`jsxSingleQuote`](https://prettier.io/docs/en/options#jsx-quotes) with the intent of matching the [`jsx-quotes`](https://eslint.org/docs/latest/rules/jsx-quotes) rule defined in [`standard/eslint-config-standard-jsx/eslintrc.json#L24`](https://github.com/standard/eslint-config-standard-jsx/blob/b21a96c35092f7e70195e5a58c410fc981d2830b/eslintrc.json#L24)

- the current value behaves more like `"prefer-double"`
- the new value behaves more like `"prefer-single"`

Background: When `prettier-config-standard` was first created, I made the call to treat JSX quotes more like HTML attributes (which I typically double quoted) 
</details>